File: All Languages/C++ Codes/FizzBuzz.cpp
vector<string> solve(int n) {
   vector<string> result;
        
        for(int i=1;i<=n;i++)
        {
            if( i % 15 == 0 )
            {
                result.push_back("FizzBuzz");
            }
            else if( i % 3 == 0)
            {
                result.push_back("Fizz");
            }
            else if ( i % 5 == 0) 
            {
                result.push_back("Buzz");
            } 
            else
            {
                result.push_back(to_string(i));
            }
        }
        return result;
}

File: All Languages/Python Codes/fizzbuzz.py
n = int(input("Enter a number"))
for i in range(n+1):
    if i%3 == 0 :
        if i%5 == 0 :
            print("FizzBuzz")
        else:
            print("Fizz")
    elif i%5==0 :
        print("Buzz")
    else:
        print(i)